Assets/Scripts/Utilities/Utilities.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TankBattleUtilities
```

Comment:
use the name "Extension" in the class name - standard c# convention of naming ext classes

Reply:
instead of Utilities and make the class as well static

Reply:
Has made the class static. But it's not an extension method. It's just a function for generating a Vector3 point.